lxd/db/cluster/identities.go

```diff
@@ -462,6 +462,41 @@ func GetPendingTLSIdentityByTokenSecret(ctx context.Context, tx *sql.Tx, secret
 	return &identities[0], nil
 }
 
+// ActivateClusterLinkIdentity updates a cluster link identity to make it valid by adding the fingerprint, PEM encoded certificate, and setting
+// the type to api.IdentityTypeClusterLink.
+func ActivateClusterLinkIdentity(ctx context.Context, tx *sql.Tx, identifier uuid.UUID, cert *x509.Certificate) error {
```

Comment:
Discussed in 1:1 - Factor out the identity type from `ActivateTLSIdentity` and use that instead